src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2390,6 +2397,15 @@ public function attributesToArray()
 	{
 		$attributes = $this->getArrayableAttributes();
 
+		// If an attribute is listed as to "encrypt", we'll decrypt it using Crypt facade.
+		foreach ($attributes as $key => $value)
+        {
```

Comment:
Let me check, I'm using sublime + smartgit and the identation was fine in both tools.